src/tracks/SampleTrack.cpp

```diff
@@ -321,6 +336,36 @@ void SampleTCO::loadSettings( const QDomElement & _this )
 	if (_this.hasAttribute("sample_rate")) {
 		m_sampleBuffer->setSampleRate(_this.attribute("sample_rate").toInt());
 	}
+
+	if (_this.hasAttribute("leftP1x") && _this.hasAttribute("leftP1y") &&
+	    _this.hasAttribute("leftP2x") && _this.hasAttribute("leftP2y"))
+	{
+		m_sampleBuffer->setLeftP1(_this.attribute("leftP1x").toFloat(),
+		                          _this.attribute("leftP1y").toFloat());
+		m_sampleBuffer->setLeftP2(_this.attribute("leftP2x").toFloat(),
+		                          _this.attribute("leftP2y").toFloat());
```

Comment:
Do we want to align with spaces? Or just split parameters out with a single tab? I feel like a single tab is a lot easier to do than holding space for 26 characters.
```suggestion
		m_sampleBuffer->setLeftP1(
			_this.attribute("leftP1x").toFloat(),
			_this.attribute("leftP1y").toFloat()
		);
		m_sampleBuffer->setLeftP2(
			_this.attribute("leftP2x").toFloat(),
			_this.attribute("leftP2y").toFloat()
		);
```

Reply:
I also think the second one is better here. For function declarations though I think the first one is easier on the eyes.